pkgs/by-name/sq/sqldef/package.nix

```
@@ -2,29 +2,29 @@
 
 buildGoModule rec {
   pname = "sqldef";
-  version = "0.17.20";
+  version = "0.17.23";
 
   src = fetchFromGitHub {
     owner = "k0kubun";
```

```suggestion
    owner = "sqldef";
```

and please test this with `nix-build . -A sqldef.src --check`